benchmark/benchmarks.jl

```diff
@@ -85,15 +85,15 @@ function fitbobyqa(dsnm::Symbol, i::Integer)
     return fit(MixedModel, fms[dsnm][i], dataset(dsnm); contrasts, progress=false)
 end
 
-SUITE["simplescalar"] = BenchmarkGroup(["single", "simple", "scalar"])
-for (ds, i) in [
-    (:dyestuff, 1),
-    (:dyestuff2, 1),
-    (:pastes, 1),
-    (:sleepstudy, 1),
-]
-    SUITE["simplescalar"][string(ds, ':', i)] = @benchmarkable fitbobyqa($ds, $i)
-end
+# SUITE["simplescalar"] = BenchmarkGroup(["single", "simple", "scalar"])
```

Comment:
Could you add a comment about why this is commented out so somebody in the future can uncomment as applicable?

Reply:
```suggestion
# these tests are so fast that they can be very noisy because the denominator is so small, 
# so we disable them by default for auto-benchmarking
# SUITE["simplescalar"] = BenchmarkGroup(["single", "simple", "scalar"])
```